plugins/examples/app-menus/package.json

```diff
@@ -30,6 +30,6 @@
     ]
   },
   "devDependencies": {
-    "@kinvolk/headlamp-plugin": "^0.8.0-alpha.3"
+    "@kinvolk/headlamp-plugin": "file:../../headlamp-plugin/kinvolk-headlamp-plugin-0.9.0-alpha.1.tgz"
```

Comment:
I've merged it into another branch so I can tweak these dependency lines after a release of headlamp-plugin.